src/dropdown.ts

```diff
@@ -1512,69 +1510,65 @@ export default class Dropdown extends LitElement implements FormControl {
       this.isShowSingleSelectIcon = Boolean(
         this.lastSelectedAndEnabledOption?.value,
       );
-    }
 
-    // Dropdown becomes filterable if there are more than 10 options. But input field
-    // won't have rendered yet given we just set `this.isFilterable` above. So we
-    // piggyback off of `this.requestUpdate()` and then wait for the update to complete
-    // before setting the `value` of input field.
-    await this.updateComplete;
+      // Dropdown becomes filterable if there are more than 10 options. But input field
+      // won't have rendered yet given we just set `this.isFilterable` above. So we wait
+      // for it to render (or not) before setting the value of input field.
+      await this.updateComplete;
 
-    if (!this.multiple) {
-      for (const option of this.#optionElements) {
-        if (option.selected) {
-          // When Dropdown is single-select, a Dropdown Option only appears as selected when
-          // it's the last selected option because only the `value` of the last selected option
-          // will be included in Dropdown's `value`. And what the user sees as selected should
-          // always be the same as what's submitted with the form.
-          //
-          // Dropdown Options determine whether they're the last selected option (and thus
-          // whether to show themselves as selected using a checkmark) via their internal
-          // `lastSelectedAndEnabledOption` getter.
-          //
-          // An additional selected option can be added to Dropdown's default slot at any time.
-          // And it may now be the last selected option. But what was the last selected option
-          // won't know it's no longer the last. So we force selected options to rerender.
-          option.requestUpdate();
-        }
-      }
-    }
+      if (
+        this.#inputElementRef.value &&
+        this.lastSelectedAndEnabledOption?.label
+      ) {
+        this.#inputElementRef.value.value =
+          this.lastSelectedAndEnabledOption.label;
 
-    if (
```

Comment:
Removed the `!multiple` condition and moved these conditionals to the `else` conditional for single-select above.